aloscene/depth.py

```diff
@@ -19,21 +21,90 @@ class Depth(aloscene.tensors.SpatialAugmentedTensor):
         loaded Depth tensor or path to the Depth file from which Depth will be loaded.
     occlusion : aloscene.Mask
         Occlusion mask for this Depth map. Default value : None.
+    is_inverse: bool
+        Either the depth is already or not inverted.
+    scale: float
+        Scale used to to shift depth. Pass this argument only if is_inverse is set to True
+    shift: float
+        Intercept used to shift depth. Pass this argument only if is_inverse is set to True
     """
 
     @staticmethod
-    def __new__(cls, x, occlusion: Mask = None, *args, names=("C", "H", "W"), **kwargs):
+    def __new__(
+            cls, 
+            x, 
+            occlusion: Mask = None, 
+            is_inverse=False, 
+            scale=None, 
+            shift=None, 
+            *args, 
+            names=("C", "H", "W"), 
+            **kwargs):
+        if is_inverse and not (shift and scale):
+            raise AttributeError('inversed depth requires shift and scale arguments')
         if isinstance(x, str):
             x = load_depth(x)
             names = ("C", "H", "W")
         tensor = super().__new__(cls, x, *args, names=names, **kwargs)
         tensor.add_child("occlusion", occlusion, align_dim=["B", "T"], mergeable=True)
-
+        tensor.add_property('scale', scale)
+        tensor.add_property('shift', shift)
+        tensor.add_property('is_inverse', is_inverse)
         return tensor
 
     def __init__(self, x, *args, **kwargs):
         super().__init__(x)
 
+    def encode_inverse(self, scale=1, shift=0):
+        """Scales and shifts the inverse depth
+        
+        Parameters
+        ----------
+            scale: (: float)
+                Multiplication factor. Default is 1.
+            
+            shift: (: float)
+                Addition intercept. Default is 0.
+        
+        Exemples
+        -------
+        >>> # Create a non inverted depth
+        >>> depth = aloscene.Depth(np.random.normal((1, 20, 20)))
+        >>> inverted_depth = depth.encode_inverse(scale=0.03, shift=1)
+        >>> depth.is_inverse, inverted_depth.is_inverse
+        >>> False, True
+        """
+        depth = self
+        if depth.is_inverse:
+            depth = self.encode_absolute()
+        depth.scale = scale
+        depth.shift = shift
+        depth.is_inverse = True
+        depth = depth * scale + shift
+        return 1 / depth
+    
+    def encode_absolute(self):
+        """Encodes inverted depth to absolute depth
+        
+        Exemples
+        --------
+        >>> # Create an already inverted depth
+        >>> invert_depth = aloscene.Depth(np.random.normal((1, 20, 20)), scale=0.5, shift=1)
+        >>> depth = depth.encode_absolute(scale=0.03, shift=1)
+        >>> depth.is_inverse, inverted_depth.is_inverse
+        >>> False, True
+        """
+        depth = self
+        if not depth.is_inverse:
+            warnings.warn('depth already encoded in absolute mode')
+            return depth
```

Comment:
return depth.clone() so that if the user use depth.encode_absolute() the returned depth will be a new tensor